packages/components/organisms/f-registration/src/components/Registration.vue

```diff
@@ -447,29 +460,27 @@ export default {
                 await RegistrationServiceApi.createAccount(this.createAccountUrl, registrationData, this.tenant);
                 this.$emit(EventNames.CreateAccountSuccess);
             } catch (error) {
-                let thrownErrors = error;
-                if (error && error.response && error.response.data && error.response.data.errors) {
-                    thrownErrors = error.response.data.errors;
-                }
-                let shouldEmitCreateAccountFailure = true;
+                if (error.response && error.response.status) {
+                    if (error.response.status === 409) {
+                        this.conflictedEmailAddress = this.email;
+                        this.$emit(EventNames.CreateAccountFailure, error);
+                        return;
+                    }
 
-                if (Array.isArray(thrownErrors)) {
-                    if (thrownErrors.some(thrownError => thrownError.errorCode === '409')) {
-                        this.shouldShowEmailAlreadyExistsError = true;
-                    } else if (thrownErrors.some(thrownError => thrownError.errorCode === 'FailedUserAuthentication')) {
-                        this.$emit(EventNames.LoginBlocked);
+                    if (error.response.status === 400) {
+                        this.genericErrorMessage = error.response.data.errors[0].description;
```

Comment:
Do we always get the `description` in a `400` error?

Reply:
Yup if its a 400 there is always an array of errors which includes a status code and description.